assets/base.css

```diff
@@ -2216,6 +2216,7 @@ input[type='checkbox'] {
 
 /* section-header */
 #shopify-section-header {
+  position: sticky;
```

Comment:
I think it might be better to apply it on `.section-header` 🤔 so it's over specific. But otherwise seems to do the trick 👍

Reply:
Because it's one of the first thing to load on the page. It's considered critical styling so it's faster this way. Since it's right there vs doing a network request for a `.css` file and waiting for it to be downloaded and parsed which can depend on the quality of your connection. ⚡